package/kernel/linux/modules/other.mk

```diff
@@ -1230,3 +1230,36 @@ define KernelPackage/it87-wdt/description
 endef
 
 $(eval $(call KernelPackage,it87-wdt))
+
+
+define KernelPackage/serdev-core
+  SUBMENU:=$(OTHER_MENU)
+  TITLE:=Serial device bus core support
+  DEPENDS:=@LINUX_4_19
+  KCONFIG:=CONFIG_SERIAL_DEV_BUS
+ifneq ($(wildcard $(LINUX_DIR)/drivers/tty/serdev/core.ko),)
```

Comment:
Why is this ifneq needed here?

Reply:
Because the package might be empty when `CONFIG_SERIAL_DEV_CTRL_TTYPORT=y` is selected. This requires `CONFIG_SERIAL_DEV_BUS=y` and that means that this functionality is built-in to the kernel. At least that's how I understood it so far. But I'm happy I you could teach me here :smile:

Reply:
The build system is able to detect built-in modules and just skips the files without any errors, so no need to check for that.